code/datums/traits/negative.dm

```diff
@@ -110,17 +110,13 @@
 /datum/quirk/depression
 	name = "Depression"
 	desc = "You sometimes just hate life."
-	mob_trait = TRAIT_DEPRESSION
 	value = -1
+	mob_trait = TRAIT_DEPRESSION
 	gain_text = "<span class='danger'>You start feeling depressed.</span>"
 	lose_text = "<span class='notice'>You no longer feel depressed.</span>" //if only it were that easy!
 	medical_record_text = "Patient has a mild mood disorder causing them to experience acute episodes of depression."
 	mood_quirk = TRUE
-	hardcore_value = 1
-
-/datum/quirk/depression/on_process(delta_time)
-	if(DT_PROB(0.05, delta_time))
-		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "depression_mild", /datum/mood_event/depression_mild)
+	hardcore_value = 2
```

Comment:
Undocumented change? What does this do?

Reply:
>    Previously, the probability of the Jolly/Depression moodlet firing was
>   so low, you only had a less than 1/3 chance of seeing the 2 minute
>    moodlet in an hour of play. Now they will be far more active.
>    Depression's hardcore point value has increased, since it is far more
 >   negative now.

That would be the hardcore point value, how many hardcore points you get if you escape on the shuttle with this quirk as one of your "hardcore random negative quirks".